StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -424,108 +433,124 @@ extension LinkAccountPickerViewController: LinkAccountPickerBodyViewDelegate {
-                        if let institution = selectedPartnerAccount.institution {
-                            self.delegate?.linkAccountPickerViewController(
-                                self,
-                                requestedPartnerAuthWithInstitution: institution
-                            )
-                        } else {
-                            self.delegate?.linkAccountPickerViewController(
+                    let didSelectContinue: () -> Void = { [weak self] in
+                        guard let self else { return }
+                        if selectedPartnerAccount.nextPaneOnSelection == .partnerAuth {
+                            if let institution = selectedPartnerAccount.institution {
+                                self.delegate?.linkAccountPickerViewController(
+                                    self,
+                                    requestedPartnerAuthWithInstitution: institution
+                                )
+                            } else {
+                                self.delegate?.linkAccountPickerViewController(
+                                    self,
+                                    didRequestNextPane: .institutionPicker
+                                )
+                            }
+                        }
+                        // nextPaneOnSelection == bankAuthRepair
+                        else {
+                            dataSource
+                                .analyticsClient
+                                .logUnexpectedError(
+                                    FinancialConnectionsSheetError
+                                        .unknown(
+                                            debugDescription: "Updating a repair account, but repairs are not supported in Mobile."
+                                        ),
+                                    errorName: "UpdateRepairAccountError",
+                                    pane: .linkAccountPicker
+                                )
+                            delegate?.linkAccountPickerViewController(
                                 self,
                                 didRequestNextPane: .institutionPicker
                             )
                         }
                     }
-                    // nextPaneOnSelection == bankAuthRepair
-                    else {
-                        dataSource
-                            .analyticsClient
-                            .logUnexpectedError(
-                                FinancialConnectionsSheetError
-                                    .unknown(
-                                        debugDescription: "Updating a repair account, but repairs are not supported in Mobile."
-                                    ),
-                                errorName: "UpdateRepairAccountError",
-                                pane: .linkAccountPicker
+
+                    let genericInfoViewController = GenericInfoViewController(
```

Comment:
This is a lot easier to review if whitespaces are removed:

<img width="321" alt="Screenshot 2024-07-25 at 2 15 37 PM" src="https://github.com/user-attachments/assets/98cf5147-8cd3-48ad-b627-09b0639b0990">